examples/ensemble/plot_gradient_boosting_regression_multioutput.py

```diff
@@ -0,0 +1,60 @@
+"""
+=========================================
+Multi-output Gradient Boosting Regression
+=========================================
+
+An example to illustrate multi-output regression with gradient boosting.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression with an estimator that does not
+natively supoort it. For comparison a regression tree based model is
```

Comment:
Oh I meant it might be interesting to compare the same estimator with native multioutput support and the MultiOutput wrapper to see if there is a significant difference, just a thought :-)